stdlib/asyncio/events.pyi

```diff
@@ -56,15 +56,21 @@ class TimerHandle(Handle):
         def when(self) -> float: ...
 
 class AbstractServer:
+    @abstractmethod
     def close(self) -> None: ...
     if sys.version_info >= (3, 7):
         async def __aenter__(self: Self) -> Self: ...
         async def __aexit__(self, *exc: Any) -> None: ...
+        @abstractmethod
         def get_loop(self) -> AbstractEventLoop: ...
+        @abstractmethod
         def is_serving(self) -> bool: ...
+        @abstractmethod
         async def start_serving(self) -> None: ...
+        @abstractmethod
         async def serve_forever(self) -> None: ...
 
+    @abstractmethod
     async def wait_closed(self) -> None: ...
 
 class AbstractEventLoop(metaclass=ABCMeta):
```

Comment:
Maybe we should remove the metaclass here, and figure out why it was originally added to this class but not to `AbstractServer`.

Reply:
`AbstractEventLoop` actually does have `ABCMeta` as the metaclass at runtime, though: https://github.com/python/typeshed/blob/5081f684c0ddce5dca9cb0678583c0793bfdef35/stdlib/asyncio/events.pyi#L70

Reply:
Yes.

Reply:
`AbstractServer` was added without the metaclass in 0f5c38a2ce9e99b8bfdbffc5f22cbaa371a69873, when `AbstractEventLoop` already had the metaclass. This doesn't seem to be intentional.

Reply:
It might be because of this line, where the metaclass lie is in fact necessary, or else mypy will complain that you either need to implement all abstractmethods from the base class or use `ABCMeta` as the metaclass to be explicit about the fact that you want this class that has no abstractmethods in it to be abstract:

https://github.com/python/typeshed/blob/7fa3d9017d54764694b6c48f25729eabfe02a514/stdlib/asyncio/events.pyi#L513